### PR TITLE
Add missing shortcuts for delete word forward and backwards

### LINKED ISSUE #1467: Ctrl + Backspace in LineEdit and TextEdit is the same as Backspace
Ctrl + Backspace is a common shortcut for deleting words, but in Slint LineEdit and TextEdit, it behaves the same as Backspace by deleting one character at a time.

### PR DESCRIPTION
**Pull request by tronical**
Fixes #1467

### CODE REVIEW

**Review by ogoffart — APPROVED** [review]

**Comment by ogoffart** [comment]
Deserves a changelog entry.

File: internal/core/input.rs [hunk 1]
```diff
@@ -276,8 +276,16 @@ impl KeyEvent {
         };
 
         match keycode {
-            key_codes::Backspace => Some(TextShortcut::DeleteBackward),
-            key_codes::Delete => Some(TextShortcut::DeleteForward),
+            key_codes::Backspace => Some(if move_mod {
```
Comment (lukas-jung) [inline internal/core/input.rs:279]: The existing code does is ordered by modifier and looks into the key itself second.
this code looks into the key first and then checks the modifier.
perhaps move that to the move_mod if block?

Comment (tronical) [reply internal/core/input.rs:279]: Good idea, done in 49ea24c44a6ff2b00f007c5165d479bc3a28ddca

**Comment by Be-ing** [comment]
Thanks for the quick fix!

**Pull request merged**